src/styles.css

```diff
@@ -158,6 +158,65 @@ header {
   color: #fff;
 }
 
+/* Homepage styling */
+.homepage-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.homepage-desc {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  color: #fff;
+}
```

Comment:
Nice styling of the homepage id, Css best practices was observed.

Reply:
Thanks @sonyco-4u